### PR TITLE
TeachingPopover: Arrow Navigation & Focus border fix

### PR DESCRIPTION
**Pull request by Mitch-At-Work**
## Previous Behavior
Tabster group focus on nav container
Some focus borders had a dual outline

## New Behavior
Arrow nav focus on nav container
Focus borders have a single outline

#30804

### CODE REVIEW

File: packages/react-components/react-teaching-popover-preview/src/components/TeachingPopoverCarouselNav/useTeachingPopoverCarouselNavStyles.styles.ts [hunk 1]
```diff
@@ -1,16 +1,29 @@
...
 export const teachingPopoverCarouselNavClassNames: SlotClassNames<TeachingPopoverCarouselNavSlots> = {
   root: 'fui-TeachingPopoverCarouselNav',
 };
 
 const useStyles = makeStyles({
-  root: { display: 'flex', flexDirection: 'row', columnGap: '4px', alignItems: 'center', justifyContent: 'center' },
+  root: {
+    display: 'flex',
+    flexDirection: 'row',
+    columnGap: '4px',
```
Comment (spmonahan) [inline packages/react-components/react-teaching-popover-preview/src/components/TeachingPopoverCarouselNav/useTeachingPopoverCarouselNavStyles.styles.ts:18]: Should this be a spacing token?

File: packages/react-components/react-teaching-popover-preview/src/components/TeachingPopoverCarouselNav/useTeachingPopoverCarouselNavStyles.styles.ts [hunk 2]
```diff
@@ -1,16 +1,29 @@
...
+    alignItems: 'center',
+    justifyContent: 'center',
+    ...createCustomFocusIndicatorStyle({
+      ...shorthands.outline('2px', 'solid', tokens.colorStrokeFocus2),
```
Comment (spmonahan) [inline packages/react-components/react-teaching-popover-preview/src/components/TeachingPopoverCarouselNav/useTeachingPopoverCarouselNavStyles.styles.ts:22]: Should this be a thickness token?

File: packages/react-components/react-teaching-popover-preview/src/components/TeachingPopoverCarouselNav/useTeachingPopoverCarouselNavStyles.styles.ts [hunk 3]
```diff
@@ -1,16 +1,29 @@
...
+      ...shorthands.borderRadius('4px'),
```
Comment (spmonahan) [inline packages/react-components/react-teaching-popover-preview/src/components/TeachingPopoverCarouselNav/useTeachingPopoverCarouselNavStyles.styles.ts:23]: A border radius token?

File: packages/react-components/react-teaching-popover-preview/src/components/TeachingPopoverCarouselNavButton/useTeachingPopoverCarouselNavButton.tsx [hunk 4]
```diff
@@ -42,6 +43,14 @@ export const useTeachingPopoverCarouselNavButton_unstable = (
     [onClick, setCurrentPage, index, onPageChange],
   );
 
+  const defaultTabProps = isSelected
+    ? getTabsterAttribute({
+        focusable: { isDefault: true },
+      })
+    : {};
+
+  console.log('Default: ', defaultTabProps);
```
Comment (spmonahan) [inline packages/react-components/react-teaching-popover-preview/src/components/TeachingPopoverCarouselNavButton/useTeachingPopoverCarouselNavButton.tsx:52]:
```suggestion

```

File: packages/react-components/react-teaching-popover-preview/src/components/TeachingPopoverCarouselNavButton/useTeachingPopoverCarouselNavButton.tsx [hunk 5]
```diff
@@ -42,6 +43,14 @@ export const useTeachingPopoverCarouselNavButton_unstable = (
     [onClick, setCurrentPage, index, onPageChange],
   );
 
+  const defaultTabProps = isSelected
```
Comment (spmonahan) [inline packages/react-components/react-teaching-popover-preview/src/components/TeachingPopoverCarouselNavButton/useTeachingPopoverCarouselNavButton.tsx:46]: I think you want to use [`useTabsterAttribute()`](https://github.com/microsoft/fluentui/blob/master/packages/react-components/react-tabster/src/hooks/useTabsterAttributes.ts) here as it memoizes this behavior.

File: packages/react-components/react-teaching-popover-preview/src/components/TeachingPopoverCarouselNavButton/useTeachingPopoverCarouselNavButtonStyles.styles.ts [hunk 6]
```diff
@@ -29,7 +29,8 @@ const useStyles = makeStyles({
     ...shorthands.padding('0px'),
     ...createCustomFocusIndicatorStyle({
       ...shorthands.outline('2px', 'solid', tokens.colorStrokeFocus2),
-      ...shorthands.borderRadius(tokens.borderRadiusCircular),
+      ...shorthands.borderRadius('4px'),
```
Comment (spmonahan) [inline packages/react-components/react-teaching-popover-preview/src/components/TeachingPopoverCarouselNavButton/useTeachingPopoverCarouselNavButtonStyles.styles.ts:32]: Use a border radius token.

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/04414c3b49686c7ce51d44ffad586a9b8de15632'>04414c3b49686c7ce51d44ffad586a9b8de15632</a></sub>
<div id="bundleSizeComment9423"/>

File: packages/react-components/react-teaching-popover-preview/src/components/TeachingPopoverHeader/useTeachingPopoverHeaderStyles.styles.ts [hunk 7]
```diff
@@ -39,12 +39,13 @@ const useStyles = makeStyles({
     borderTopRightRadius: tokens.borderRadiusNone,
     borderBottomRightRadius: tokens.borderRadiusNone,
     borderRightStyle: 'none',
-    ...createCustomFocusIndicatorStyle({
-      borderTopRightRadius: tokens.borderRadiusNone,
-      borderBottomRightRadius: tokens.borderRadiusNone,
-    }),
     marginInlineStart: 'auto',
     ...shorthands.padding(tokens.spacingVerticalXS, tokens.spacingHorizontalXS),
+    ...createCustomFocusIndicatorStyle({
```
Comment (spmonahan) [inline packages/react-components/react-teaching-popover-preview/src/components/TeachingPopoverHeader/useTeachingPopoverHeaderStyles.styles.ts:44]: Use tokens for thickness and border radius.

File: packages/react-components/react-teaching-popover-preview/src/components/TeachingPopoverTitle/useTeachingPopoverTitleStyles.styles.ts [hunk 8]
```diff
@@ -38,8 +38,9 @@ const useStyles = makeStyles({
     borderBottomRightRadius: tokens.borderRadiusNone,
     borderRightStyle: 'none',
     ...createCustomFocusIndicatorStyle({
-      borderTopRightRadius: tokens.borderRadiusNone,
-      borderBottomRightRadius: tokens.borderRadiusNone,
+      ...shorthands.outline('2px', 'solid', tokens.colorStrokeFocus2),
```
Comment (spmonahan) [inline packages/react-components/react-teaching-popover-preview/src/components/TeachingPopoverTitle/useTeachingPopoverTitleStyles.styles.ts:41]: Ditto re: tokens :)

**Review by spmonahan — APPROVED** [review]

**Review by brandonthomas — APPROVED** [review]

**Pull request merged**